data/bunrodea/bunrodea missions.txt

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2023 by Amazinite
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "First Contact: Bunrodea"
+	landing
+	source
+		near "Sol" 0 100
+		not attributes "urban"
+		attributes "spaceport"
+	destination "Erabuthro"
+	clearance "Unlike before, none of the alien ships are hostile. After hailing the planet that the aliens pointed you towards, you hear an alien voice respond. Before you can think of what to say, the small device that the aliens threw at you on <origin> responds for you. The alien on the other end says something short, then cuts the feed. It seems like you've been cleared to land."
+	to offer
+		random < 10
+		or
+			"reputation: Bunrodea" != 1
+			"reputation: Bunrodea (Megasa)" != 1
+			"reputation: Bunrodea (Erabu)" != 1
+			"reputation: Bunrodea (Guard)" != -1
+	
+	on offer
+		"bunrodea: guard rep" = "reputation: Bunrodea (Guard)"
+		"bunrodea: megasa rep" = "reputation: Bunrodea (Megasa)"
+		"bunrodea: erabu rep" = "reputation: Bunrodea (Erabu)"
+		"bunrodea: normal rep" = "reputation: Bunrodea"
+		conversation
+			`When you depart your ship, you're startled by a small alien ship uncloaking right in front of you. Two elegant mantis-like aliens step off it, each holding a strange looking weapon pointed directly at you. They have four long insectoid legs and two arms. Although their arms end in what look like hands, they also have strange claw-like features that partially enclose them, extending outward from their forearms to grasp their weapons. You look around to see if there's anyone else witnessing this, but you've landed at <origin>'s main spaceport in the middle of the night, leaving you as the only human in sight.`
+			`	The aliens begin speaking to you in a language you don't understand. As one of them speaks, they sound almost robotic, as if they are speaking through a synthesizer. Taking a closer look, you spot what you can only guess are cybernetic modifications to parts of the alien's body. The other one has no such visible modifications, though.`
```

Comment:
Since you're noting one has cybernetic modifications while the other has none visible, maybe describe what the modifications are, briefly?